src/app/component/ProductCustomizableOptions/ProductCustomizableOptions.style.scss

```diff
@@ -0,0 +1,72 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+:root {
+    --additional-information-color: #2c2c2c;
+    --required-text-color: #dc6d6d;
+}
+
+.ProductCustomizableOptions {
+    &-Content {
+        .Field {
+            &-SelectWrapper {
+                max-width: 185px;
```

Comment:
Please no magic number. Why exactly this value? Add comment if there is a solid reason

Reply:
This value was from designs, created to match them, can remove but then dropdown will be much larger